tensorflow/contrib/layers/ln_kernels/layer_norm_fused_grad_op.cc

```diff
@@ -0,0 +1,356 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Certainly, if I want to put the kernel codes in `/layers/kernel`, I would have to write build rules in the `/layers/kernel/BUILD` and then link to it with deps in the `tf_custom_op_library` build rules defined in `/layers/BUILD` ([here](https://github.com/tensorflow/tensorflow/pull/6205/files#diff-853b616fe717bd914bdae461d1ea26bfR77)). The problem I ran into is that I don't know how to write the `cc_library` build rules in `/layers/kernel/BUILD` that use gpu_srcs and couldn't find examples in the source code.